charts/k8tz/values.yaml

```diff
@@ -15,7 +15,8 @@ labels: {}
 
 webhook:
   failurePolicy: Fail
-
+  tlsMinVersion: ""
+  tlsCipherSuites: ""
   certManager:
```

Comment:
@klzsysy  sorry for nitpicking but there should be a line between tlsCipherSuites and certManager

Reply:
done